src/azure-cli/azure/cli/command_modules/appservice/_validators.py

```diff
@@ -169,7 +169,8 @@ def validate_asp_sku(cmd, namespace):
 def validate_ip_address(cmd, namespace):
     if namespace.ip_address is not None:
         _validate_ip_address_format(namespace)
-        _validate_ip_address_existance(cmd, namespace)
+        if 'add' in cmd.name:
```

Comment:
Add a comment to the code here so we don't remove this by mistake. Can you also a test case for delete IP address ?? so we don't regress this.